bindings/matrix-sdk-ffi/src/platform.rs

```diff
@@ -94,36 +98,126 @@ fn text_layers<S>(config: TracingConfiguration) -> impl Layer<S>
 where
     S: Subscriber + for<'a> LookupSpan<'a>,
 {
-    fn fmt_layer<S>() -> fmt::Layer<S> {
-        fmt::layer()
-            .with_file(true)
-            .with_line_number(true)
-            .with_ansi(cfg!(not(any(target_os = "android", target_os = "ios"))))
```

Comment:
Yeah, ANSI support would have been pretty annoying because a bunch of functionality around that is private within `tracing-subscriber`, and we're not actually using it ourselves in the FFI context.